index.bs

```diff
@@ -10696,6 +10703,95 @@ The [=remote end steps=] given |session| and |command parameters| are:
 
 </div>
 
+## The autofill Module ## {#module-autofill}
+
+The <dfn export for=modules>autofill module</dfn> contains functionality for triggering autofill.
+
+### Definition ### {#module-autofill-definition}
+
+[=remote end definition=]
+
+<pre class="cddl remote-cddl">
+
+InputCommand = (
+  autofill.trigger
+)
+</pre>
+
+### Commands ### {#module-autofill-commands}
+
+#### The autofill.trigger Command ####  {#command-autofill-trigger}
+
+The <dfn export for=commands>autofill.trigger</dfn> command triggers autofill
+on a particular form field using the provided fields.
+
+<dl>
+   <dt>Command Type</dt>
+   <dd>
+    <pre class="cddl remote-cddl">
+      autofill.trigger = (
+        method: "autofill.trigger",
+        params: autofill.triggerParameters
+      )
+
+      autofill.triggerParameters = {
+        context: browsingContext.BrowsingContext,
+        element: script.SharedReference,
+        fields: [*autofill.field]
+      }
+
+      autofill.field = (
+        name: tstr,
```

Comment:
I think we would probably need a named type for the field name and a definition what that type represents (linking to the HTML spec).

Reply:
good point. I've tried my best to define a named type and link to the respective section in the autofill documentation to define what values are allowed. Would appreciate some more pair of eyes over it to make sure the type is correct and valid.